pom.xml

```diff
@@ -30,7 +30,7 @@
     <groupId>org.glassfish.build</groupId>
     <artifactId>glassfishbuild-maven-plugin</artifactId>
     <packaging>maven-plugin</packaging>
-    <version>4.0.1-SNAPSHOT</version>
+    <version>4.0.2-SNAPSHOT-SNAPSHOT</version>
```

Comment:
- :eyes: https://github.com/eclipse-ee4j/glassfish-spec-version-maven-plugin/pull/144#issuecomment-2031457337

Reply:
I have fixed that now ...